### PR TITLE
update cf-1528A.mdx

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/cf-1528A.mdx [hunk 1]
```diff
@@ -133,4 +133,72 @@ int main() {
...
+public class ParsasHumongousTree {
+	private static Kattio io = new Kattio();
+	private static List<List<Integer>> adj = new ArrayList<>();
+	private static int[] l, r;
+	private static long[][] dp;
+
+	public static void main(String[] args) {
+		int t = io.nextInt();
+
+		while (t-- != 0) {
```
Comment (SansPapyrus683) [inline solutions/cf-1528A.mdx:151]: please use a for loop

File: solutions/cf-1528A.mdx [hunk 2]
```diff
@@ -133,4 +133,72 @@ int main() {
...
+				adj.add(new ArrayList<>());
+			}
+
+			for (int i = 0; i < n - 1; i++) {
+				int u = io.nextInt(), v = io.nextInt();
+				adj.get(u).add(v);
+				adj.get(v).add(u);
+			}
+
+			dfs(1, -1);
```
Comment (SansPapyrus683) [inline solutions/cf-1528A.mdx:174]: dfs says nothing about the function name

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
please make the code consistent w/ the c++ code in names & code structure

File: solutions/cf-1528A.mdx (hunk 1 above)
Comment (876pol) [reply solutions/cf-1528A.mdx]: ok fixed

File: solutions/cf-1528A.mdx (hunk 2 above)
Comment (876pol) [reply solutions/cf-1528A.mdx:174]: i mean the function is just a dfs.. its also what the c++ code uses

Comment (SansPapyrus683) [reply solutions/cf-1528A.mdx:174]: what does it do during the dfs though?

File: solutions/cf-1528A.mdx [hunk 3]
```diff
@@ -133,4 +133,85 @@ int main() {
...
 ```
 </CPPSection>
+
+<JavaSection>
+```java
+import java.io.*;
+import java.util.*;
+
+public class ParsasHumongousTree {
+	private static Kattio io = new Kattio();
```
Comment (SansPapyrus683) [inline solutions/cf-1528A.mdx:143]: don't think this needs to be global

**Review by SansPapyrus683 — APPROVED** [review]
@envyaims

File: solutions/cf-1528A.mdx (hunk 3 above)
Comment (876pol) [reply solutions/cf-1528A.mdx:143]: ok fixed

**Review by envyaims — APPROVED** [review]
ok

File: solutions/cf-1528A.mdx [hunk 4]
```diff
@@ -126,11 +126,92 @@ int main() {
...
+
+			calculate_dp(1, -1);
+
+			io.println(Math.max(dp[1][0], dp[1][1]));
+		}
+
+		io.flush();
+	}
+
+	private static void calculate_dp(int u, int par) {
```
Comment (SansPapyrus683) [inline solutions/cf-1528A.mdx:182]: use camelCase

Comment (876pol) [reply solutions/cf-1528A.mdx]: oh oops forgot that it's java

**Pull request merged**